bin/azure/deploy

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+ROOT=$SCRIPT/../..
+
+echo "Generate SSL Certificate and Key for localhost"
+echo "--------------------------------"
+echo "Generate root key"
+echo "--------------------------------"
+openssl genrsa 2048 > ${ROOT}/conf/azure/rabbitmq-ca.key
+echo ""
+echo "--------------------------------"
+echo "Create and self-sign root certificate"
+echo "--------------------------------"
+openssl req -new -x509 -nodes -days 365 \
+   -key ${ROOT}/conf/azure/rabbitmq-ca.key \
+   -out ${ROOT}/conf/azure/rabbitmq-ca.crt \
+   -subj "/C=US/ST=California/L=San Francisco/O=RabbitMQ/OU=OAuth 2.0 Tutorial/CN=RootCA"
+echo ""
+echo "--------------------------------"
+echo "Create Certificate Signing Request and associated private key for localhost"
+echo "--------------------------------"
+openssl req -newkey rsa:2048 -nodes \
+   -keyout ${ROOT}/conf/azure/rabbitmq.key \
+   -out ${ROOT}/conf/azure/rabbitmq.csr \
+   -subj "/C=US/ST=California/L=San Francisco/O=RabbitMQ/OU=OAuth 2.0 Tutorial/CN=localhost"
+echo ""
+echo "--------------------------------"
+echo "Create certificate for localhost"
+echo "--------------------------------"
+openssl x509 -req -days 365 \
+   -in ${ROOT}/conf/azure/rabbitmq.csr \
+   -out ${ROOT}/conf/azure/rabbitmq.crt \
+   -CA ${ROOT}/conf/azure/rabbitmq-ca.crt \
+   -CAkey ${ROOT}/conf/azure/rabbitmq-ca.key \
+   -CAcreateserial
+echo ""
+echo "--------------------------------"
+echo "Configure SSL cert/key ownership"
+echo "--------------------------------"
+chown 999:999 ${ROOT}/conf/azure/rabbitmq-ca.crt \
```

Comment:
When I run it on my mac, these two lines failed. I skipped them and everything worked fine. @baptistedaroit Do we need them?

Reply:
@MarcialRosales I think it depends on how docker is configured on the machine. In my case, I needed it because the key was mounted in the container with root as owner, with permissions 600 thus preventing rabbitmq to read it at startup.